requirements.txt

```diff
@@ -16,19 +16,19 @@ django-cors-headers==2.0.2
 # crypto
 pycryptodome==3.6.5
 rsa==3.4.2
-merkletools==1.0.2
-pysha3==1.0.2  #1.0b1
+merkletools>=1.0.2
+pysha3>=1.0.2  #1.0b1
```

Comment:
This lib `pysha3` plus `blockcypher` had conflicts, this update fix the `python pip install -r requirements.txt` @ebarojas

Reply:
I don't think we still use blockcypher.